addons/account/tests/test_account_move_entry.py

```diff
@@ -530,6 +533,54 @@ def test_journal_sequence_ordering(self):
         self.assertEqual(copies[5].name, 'XMISC/2019/10005')
         self.assertEqual(copies[5].state, 'draft')
 
+    def test_sequence_concurency(self):
+        with db_connect(self.env.registry.db_name).cursor() as cr0,\
+                db_connect(self.env.registry.db_name).cursor() as cr1,\
+                db_connect(self.env.registry.db_name).cursor() as cr2:
+            env0 = api.Environment(cr0, SUPERUSER_ID, {})
+            env1 = api.Environment(cr1, SUPERUSER_ID, {})
+            env2 = api.Environment(cr2, SUPERUSER_ID, {})
+
+            journal = env0['account.journal'].create({
+                'name': 'concurency_test',
+                'code': 'CT',
+                'type': 'general',
+            })
+            account = env0['account.account'].create({
+                'code': 'CT',
+                'name': 'CT',
+                'user_type_id': env0.ref('account.data_account_type_fixed_assets').id,
+            })
+            move_ids = env0['account.move'].create([{
```

Comment:
This name `move_ids` got me confused below.
Those are not move ids, those are moves, call them `moves`.

Reply:
Meh, the general naming convention in Odoo is bad IMO.
```python
record.related_ids  # this is not a set of ids either
record.related_ids.ids  # doesn't this look stupid?
```

I guess we chose that because it is the ids that are stored in the db, but... I mean, what else is it gonna be in the column `account` than the account's id?

But ok, I'll change it. I like it better too.